vmdb/spec/requests/api/tag_collections_spec.rb

```diff
@@ -29,441 +43,361 @@ def app
   end
 
   context "Provider Tag subcollection" do
-    before(:each) do
-      @provider          = @ems
-      @provider_url      = "#{@cfme[:providers_url]}/#{@ems.id}"
-      @provider_tags_url = "#{@provider_url}/tags"
+    let(:provider)          { ems }
+    let(:provider_url)      { providers_url(provider.id) }
+    let(:provider_tags_url) { "#{provider_url}/tags" }
```

Comment:
and for some reason, I like having a let for `provider` or `ems`, but not sure we need it for `provider_url(ems)`. or `provider_tags_url(ems)`